NOTES.md

```diff
@@ -18,15 +18,16 @@ This file documents changes to Argus-frontend that are important for the users t
 - Logo and favicon
 - Add seconds to timestamps in elements of the event feed in detailed incident view
 
+- Option "None" is default in the phone number selector in notification profiles.
+
 - Order of events and acknowledgements in feed in detailed incident view. Order is now oldest-first.
 
 
 
 ### Added
 - End time to the incident detailed view (for closed incidents only)
 
-
-
+- Option "None" to the phone number selector in notification profiles.
```

Comment:
Describe the point of the change instead: 'Make it easier/more obvious to save a profile without a phone number by selecting "None"' or something like that. This isn't the first line of a commit-message, it's okay to be human :)